data-browser/src/main/scala/hmda/data/browser/models/ModifiedLarEntity.scala

```diff
@@ -0,0 +1,237 @@
+package hmda.data.browser.models
+
+import java.sql.Timestamp
+
+import slick.jdbc.GetResult
+
+case class ModifiedLarEntity(
```

Comment:
Do we need to copy the modified lar entity and repository in the data-browser or can we just pull those from common?

Reply:
https://github.com/cfpb/hmda-platform/issues/2739